simpler_eda/numerical_eda.py

```diff
@@ -80,5 +74,146 @@ def numerical_plot(
                  title = " Horsepower vs Acceleration",
                  font_size = 10)
     """
-
-    return numerical_plot
+    # Defensive programming: Check if user provides valid inputs
+
+    # Ensure title variable is specified by the user
+    assert title is not None, "Variable title needs to be specified."
+
+    error_one = "TypeError: Data must be entered as a pandas dataframe."
+    assert isinstance(data, pd.DataFrame), error_one
+
+    error_two = "Wrong type! X-axis variable must be entered as a String."
```

Comment:
Shall we use either "TypeError" or "Wrong type!" for all wrong types? Or It's intended?

Reply:
They are the same idea, but I will change it to TypeError to be consistent